__tests__/endpoints.test.js

```diff
@@ -161,6 +163,29 @@ describe("GET /api/articles/1/comments", () => {
   });
 });
 
+describe("GET articles filtered by topic", () => {
+  test.only("GET 200 responds with articles filtered by topic", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then((response) => {
+        const { articles } = response.body;
+        articles.forEach((article) => {
+          expect(article.topic).toBe("mitch");
+          expect(articles).toHaveLength(12);
+        });
+      });
+  });
+  test.only("GET 404 responds with a 404 status code if no articles are found for the given topic", () => {
+    return request(app)
+      .get("/articles?topic=not-a-topic")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("Not Found");
+      });
+  });
+});
```

Comment:
Nice 👍 

Pleas make sure to remove .onlys from your tests before pushing :) 

We need just one more test here:

- 200: Topic exists but has no articles. Should return an empty array